addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -141,15 +140,27 @@
                     <t t-else="">
                         <t t-if="forum.mode == 'questions'">
                             <a t-attf-href="?#{ keep_query('search', 'sorting', 'my', filters='solved') }"
-                               class="dropdown-item">Solved
+                                class="dropdown-item">Solved
                             </a>
                             <a t-attf-href="?#{ keep_query('search', 'sorting', 'my', filters='unsolved') }"
-                               class="dropdown-item">Unsolved
+                                class="dropdown-item">Unsolved
                             </a>
                         </t>
                         <a t-attf-href="?#{ keep_query('search', 'sorting', 'my', filters='unanswered') }"
-                           class="dropdown-item">Unanswered
+                            class="dropdown-item">Unanswered
                         </a>
+                        <t t-if="user.karma>=forum.karma_moderate">
+                            <div class="dropdown-divider"/>
+                            <a t-attf-href="/forum/#{slug(forum)}/validation_queue"
```

Comment:
It's a detail, but could you keep please the spaces in `#{ something }` as we did in this file?